app/mailers/user_mailer.rb

```diff
@@ -293,6 +293,17 @@ def admin_deleted_work_notification(user, work)
     )
   end
 
+  # Sends email to authors when a creation is hidden by an Admin
+  def admin_hidden_work_notification(creation,user)
```

Comment:
What sarken is saying is unleash the hound.

( I would do a pull first to get the puppy control )